dask/delayed.py

```diff
@@ -246,28 +249,68 @@ def _inner(self, other):
     return _inner
 
 
+class DelayedFunction(base.Base):
+    _optimize = staticmethod(lambda dsk, keys, **kwargs: dsk)
+    _finalize = staticmethod(first)
+    _default_get = staticmethod(get_sync)
```

Comment:
Seems inevitable though, right?  Is threaded or multiprocessing more appropriate here?  I would claim that neither is particularly more accurate.  Might as well go with what makes more sense for this object and, in the rare case that this gets called, suffer the informative error.